webapp/src/tsconfig.app.json

```diff
@@ -1,15 +1,18 @@
 {
-  "extends": "../tsconfig.json",
-  "compilerOptions": {
-    "outDir": "../out-tsc/app",
-    "types": [],
-    "module": "esnext",
-  },
-  "files": [
-    "main.ts",
-    "polyfills.ts"
-  ],
-  "include": [
-    "src/**/*.d.ts"
-  ]
-}
+	"extends": "../tsconfig.json",
+	"compilerOptions": {
+		"outDir": "../out-tsc/app",
+		"types": [],
+		"module": "esnext",
+	},
+	"files": [
+		"main.ts",
+		"polyfills.ts"
+	],
+	"include": [
+		"src/**/*.d.ts"
+	],
+	"angularCompilerOptions": {
+		"enableIvy": false
```

Comment:
@rahul-rathore-576 did you try to enable Ivy? It did not work? Too many changes required? Asking not for this PR specifically (it's fine to merge with Ivy disabled), but for the future to understand how much effort we need to do to enable Ivy?

Reply:
@evereq I have tried with enabled Ivy and its working fine.